src/Components/AuthInput/index.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './styles.module.scss';
+import { handleAuthInputStyles } from '../../utils';
+
+
+export const AuthInput = React.memo(function AuthInput({
```

Comment:
`.memo` wrapper without `areEqual` function provided will perform shallow comparison, so when `inputData` changes, you'll have component re-rendered. Is it done intentionally?